tests/kw_env_test.sh

```diff
@@ -219,4 +219,64 @@ function test_parse_env_options()
   assertEquals "($LINENO) Invalid option" "$?" 22
 }
 
+function test_destroy_env_checking_output_message()
+{
+  local output
+  local expected_result
+
+  local -a inputs=(
+    'y'
+    'n'
+  )
+
+  # Create envs
+  options_values['CREATE']='X86_64_MACHINE_A'
+  create_new_env
+
+  options_values['CREATE']='X86_64_MACHINE_B'
+  create_new_env
+
+  # Destroy an env
+  options_values['DESTROY']='X86_64_MACHINE_A'
+  output="$(printf '%s\n' "${inputs[0]}" | destroy_env)"
+
+  expected_result='The X86_64_MACHINE_A environment has been destroyed.'
+
+  assertEquals "($LINENO)" "$expected_result" "$output"
```

Comment:
It is better to drop this test because you are checking the user string, which is a little fragile since we can change the user message in the code.

Reply:
you are right, i'll remove that test.